### PR TITLE
Feat/message mvc

### PR DESCRIPTION
**Pull request by fredl2tp**
MVC feature message, fix message table, recipient table with data.

### CODE REVIEW

File: backend/database.sql [hunk 1]
```diff
@@ -215,4 +217,52 @@ VALUES
...
+("Titre 13", "Contenu 13", "2023-02-02", "Non Lu", 13),
+("Titre 14", "Contenu 14", "2023-02-02", "Lu", 14),
+("Titre 15", "Contenu 15", "2023-02-02", "Non Lu", 15),
+("Titre 16", "Contenu 16", "2023-02-02", "Lu", 16),
+("Titre 17", "Contenu 17", "2023-02-02", "Lu", 17),
+("Titre 18", "Contenu 18", "2023-02-02", "Non Lu", 18),
+("Titre 19", "Contenu 19", "2023-02-02", "Lu", 19),
+("Titre 20", "Contenu 20", "2023-02-02", "Lu", 20);
+
+-- Query SQL user_message
```
Comment (Christophe-76) [inline backend/database.sql]: recipient

**Review by Christophe-76 — APPROVED** [review]
the tables have been checked, it's good

**Review by RolandEsnay — APPROVED** [review]
check done, all is good

**Review by Christophe-76 — APPROVED** [review]
change ok, table message ok

**Review by Eric-Rouviere — APPROVED** [review]
Check ok Please rename CONSTRAINT fk_author_id to fk_user_author_id ?

Rename recipient_id in recipient table into user_id ?

**Comment by fredl2tp** [comment]
> Check ok Please rename CONSTRAINT fk_author_id to fk_user_author_id ?
> 
> Rename recipient_id in recipient table into user_id ?

No Eric. The tables conform to the Data Logic Model. We need to first discuss it and incorporate it into our modeling.

**Pull request merged**